scenes/cslib_number.txt

```diff
@@ -1,3 +1,5 @@
+*bug "cslib_number is a cslib module and must not be accessed directly"
```

Comment:
Sorry, same again here, please could you remove the quotes?

Reply:
FYI, I believe you can do it too.